Wikipedia/Code/WMFTableViewCellWithCustomMinusButton.m

```diff
@@ -0,0 +1,71 @@
+#import "WMFTableViewCellWithCustomMinusButton.h"
+#import <Masonry/Masonry.h>
+
+@interface WMFTableViewCellWithCustomMinusButton ()
+
+@property (strong, nonatomic) UIButton* minusButton;
+@property (strong, nonatomic) CALayer* cellWhiteLayer;
+@property (strong, nonatomic) CALayer* minusButtonWhiteLayer;
+
+@end
+
+@implementation WMFTableViewCellWithCustomMinusButton
+
+- (void)layoutSubviews {
+    [super layoutSubviews];
+    
+    // HAX: ensure the custom button is always on top.
+    [self.minusButton.superview bringSubviewToFront:self.minusButton];
+
+    self.cellWhiteLayer.frame = self.bounds;
+    self.minusButtonWhiteLayer.frame = self.minusButton.bounds;
+}
+
+- (void)awakeFromNib {
+    [super awakeFromNib];
+    
+    self.minusButton = [self wmf_minusButton];
+    [self.contentView.superview addSubview:self.minusButton];
+    [self.minusButton mas_makeConstraints:^(MASConstraintMaker* make) {
+        make.width.equalTo(@(50));
+
+        // HAX: attach leading edge to contentView so it tracks horizonally
```

Comment:
Not a hack - good to comment… but aligning a view to another view isn't a hack. We should reserve this word for things that are actually hacks.